luaui/Widgets_BAR/cmd_select_commander.lua

```diff
@@ -0,0 +1,28 @@
+function widget:GetInfo()
+  return {
+    name = "SelectCommander",
+    desc = "Press ctrl+c to select the commander",
+    author = "Teifion",
+    date = "26/10/2020",
+    license = "GNU GPL, v2 or later",
+    layer = 0,
+    enabled = true
+  }
+end
+
+function widget:Update()
+  local t = Spring.GetGameSeconds()
+  if (select(3,Spring.GetPlayerInfo(Spring.GetMyPlayerID(),false)) or t > 10) then
```

Comment:
Why remove if `t > 10`? Should doing `/luaui reload` remove the ability to ues Ctrl+C?

Reply:
I left it in because it was used in select_and_center and I thought it should be removed but wanted to leave it in incase I misunderstood something, I just posted about it on the dev channel. I'd be happy to remove it.

I was not aware of GetSpectatingState, that would be a much easier way to do it.